authentication/using-tokens.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Using Client Tokens
```

Comment:
what about just merging this page under the `Token` section in the `Method` page above?

Reply:
My thought process here was that "client tokens" aren't strictly tied to a single method.
`token` is obviously the most likely method to leverage this, given it is just a static instantiation of a client token.

However, OIDC establishes a client token in the same way and the result can be used via both means too (`Authorization` or `Cookie` header). 

What do you think? This probably doesn't provide much practical value at this stage. We might add another method in the future which could leverage this.

Reply:
yeah thats a good point, fine to keep as is. I'm just thinking that when I saw `Tokens` and `Client Tokens` they seemed like they should be under the same section. Maybe we need to think about how we differentiate the two more? 

Maybe changing the notion of `Token` (created via the API/bootstrap) to `Static Token` would make it more clear? (http://localhost:3000/authentication/methods#token).

Something we do think about/do in a separate PR tho as this one is g2g IMO!

Reply:
💯 I think you're right. The `token` method should be clearly titled `Static Tokens` to help differentiate how the token is acquired from using it on the wire.